ql/src/test/org/apache/hadoop/hive/metastore/txn/TestCompactionTxnHandler.java

```diff
@@ -742,12 +740,18 @@ public void testRevokeTimedOutWorkers() throws Exception {
   public void testFindPotentialCompactions() throws Exception {
     // Test that committing unlocks
     long txnid = openTxn();
-
-    List<LockComponent> components = new ArrayList<>();
-
-    components.add(createLockComponent(LockType.SHARED_WRITE, LockLevel.DB, "mydb", "mytable", null, DataOperationType.UPDATE));
-    components.add(createLockComponent(LockType.SHARED_WRITE, LockLevel.DB, "mydb", "yourtable", "mypartition=myvalue", DataOperationType.UPDATE));
-
+    LockComponent comp = new LockComponent(LockType.SHARED_WRITE, LockLevel.DB,
```

Comment:
why is this change? how is this related to the JIRA?

Reply:
This is a merge issue, I'll fix it.

Reply:
fixed